modern/.eslintrc.js

```diff
@@ -16,14 +14,35 @@ module.exports = {
     'no-nested-ternary': [0],
     'operator-linebreak': [0],
     'import/no-unresolved': [0],
-    'object-curly-newline': [1, {
-      ObjectExpression: { minProperties: 8, multiline: true, consistent: true },
-      ObjectPattern: { minProperties: 8, multiline: true, consistent: true },
-      ImportDeclaration: { minProperties: 4, multiline: true, consistent: true },
-      ExportDeclaration: { minProperties: 4, multiline: true, consistent: true },
-    }],
+    'object-curly-newline': [
+      1,
+      {
+        ObjectExpression: {
+          minProperties: 8,
+          multiline: true,
+          consistent: true,
+        },
+        ObjectPattern: { minProperties: 8, multiline: true, consistent: true },
```

Comment:
Please make formatting consistent here.

Reply:
Done, please check this part. I do not fully understand your meaning.